service/templates/admin.html

```diff
@@ -1906,7 +1902,7 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<input type="text" ng-model="Settings.providers.aws.task_sync_read.secret">
 									</md-input-container>
 									<md-input-container class="control" style="width:45%;">
-										<label>Task Read-Only S3 Bucket</label>
+										 must be specified<label>Task Read-Only S3 Bucket</label>
```

Comment:
this seems like it was meant to be somewhere else

Reply:
Yes thank you!